airbyte-integrations/connectors/source-google-analytics-data-api/unit_tests/test_streams.py

```diff
@@ -89,7 +89,7 @@ def test_request_body_json(patch_base_class):
         "dateRanges": [request_body_params["stream_slice"]],
         "returnPropertyQuota": True,
         "offset": str(0),
-        "limit": str(PAGE_SIZE),
+        "limit": str(GoogleAnalyticsDataApiBaseStream.PAGE_SIZE),
```

Comment:
can we add an additional test that uses this same structure, but after creating the `GoogleAnalyticsDataApiBaseStream` instance, we call `stream.page_size()` = 100, and then verify when we call `request_body_json()` that the value is honored in the return of the method?

Reply:
done